### PR TITLE
label many warps and rooms

### PR DESCRIPTION
**Pull request by EpochFlame**
room labeling and minor typo fixes/annotation

### CODE REVIEW

**Comment by EpochFlame** [comment]
rooms in dungeons now reflect the dungeon in the room name

**Review by tobiasvl — APPROVED** [review]

File: src/code/entities/bank36.asm [hunk 1]
```diff
@@ -1752,39 +1752,39 @@ Data_036_4A14::
...
-    dw   Data_036_4A39
+HintRoomTable::
+    dw   TailCaveHintRooms
+    dw   BottleGrottoHintRooms
+    dw   KeyCavernHintRooms
+    dw   AnglersTunnelHintRooms
+    dw   CatfishsMawHintRooms
+    dw   FaceShrineHintRooms
+    dw   EaglesTowerHintRooms
+    dw   TurtleRockHintRooms
```
Comment (kemenaran) [inline src/code/entities/bank36.asm:1787]: Nice!

**Review by kemenaran — APPROVED** [review]
Nice, cool work! It makes easier to understand what many code segments are about.

File: src/code/entities/bank36.asm (hunk 1 above)
Comment (EpochFlame) [reply src/code/entities/bank36.asm:1787]: Happy to help!

**Pull request merged**